sdks/python/README.md

```diff
@@ -118,7 +118,7 @@ set up your machine's Python development environment.
 #### Install ``pip``
 
 `pip` is Python's package manager.  If you already have `pip` installed
-(type `pip -V` to check), skip this step.
+(type `pip -V` to check), please make sure to have at least version 7.0.0.
```

Comment:
You are totally right! I am sorry, this PR was really long for no reason! I promise next time it will be better :)